robotiq_hande_driver/include/communication.hpp

```diff
@@ -104,7 +144,10 @@ class Communication{
      * @param value bool value
      * @return none
      */
-    void write_action_bit(uint8_t position_bit, bool value);
+    void write_action_bit(uint8_t position_bit, bool value) {
+        output_bytes_[(uint)OutputBytes::ACTION_REQUEST] = bit_set_to(
+            output_bytes_[(uint)OutputBytes::ACTION_REQUEST], position_bit, value);
```

Comment:
https://stackoverflow.com/a/1255015
```suggestion
        output_bytes_[static_cast<uint>(OutputBytes::ACTION_REQUEST)] = bit_set_to(
            output_bytes_[static_cast<uint>(OutputBytes::ACTION_REQUEST)], position_bit, value);
```